pkgs/development/python-modules/unrar2-cffi/default.nix

```
@@ -0,0 +1,63 @@
+{
+  buildPythonPackage,
+  cffi,
+  fetchPypi,
+  lib,
+  nix-update-script,
+  pytest,
+  pythonOlder,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "unrar2-cffi";
+  version = "0.4.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "unrar2_cffi";
+    hash = "sha256-P+MXSmoMiIkk42MEYoJq9Okd1JTVwEPLCjGGXrnpyDM=";
+  };
+
+  build-system = [
+    cffi
+    setuptools
+  ];
+
+  dependencies = [
+    cffi
+  ];
+
+  nativeCheckInputs = [
+    pytest
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    pytest
+
+    runHook postCheck
+  '';
```

```suggestion
  nativeCheckInputs = [
    pytestCheckHook
  ];

  preCheck = ''
    rm -r unrar
  '';
```
Another case of #123561. This works.